.travis.yml

```diff
@@ -1,4 +1,7 @@
-dist: xenial
+dist: bionic
+arch:
+- amd64
+- ppc64le
```

Comment:
This is a change that we should make in all the repos before merging this in here.

Reply:
@carbonin thank you and acknowledged, I have had to touch mostly the container-ruby and manageiq-pods repo only besides this, would you be able to point to the other repositories that need to have Travis support enabled or I guess I can look up each one under the ManageIQ umbrella? also as mentioned I am working directly with the Travis team to get the PostgreSQL sorted out on Power, so we post an update here once that is done and I can get the job to pass.